src/test/java/org/opennms/minion/stests/SyslogTest.java

```diff
@@ -84,6 +84,12 @@ public void canReceiveSyslogMessages() throws Exception {
             pipe.println("config:update");
             // Install the syslog handler feature
             pipe.println("features:install opennms-syslogd-handler-default");
+            // Point the trap handler at the local ActiveMQ broker
+            pipe.println("config:edit org.opennms.netmgt.trapd.handler.default");
+            pipe.println("config:propset brokerUri tcp://127.0.0.1:61616");
+            pipe.println("config:update");
+            // Install the trap handler feature
+            pipe.println("features:install opennms-trapd-handler-default");
```

Comment:
I guess the point I was trying to make is that the Syslogd test or the Trapd test should only be concerned about testing their respective features, and not worry about installing other features that may or may not interfere with them.

I'd rather see us introduce a "meta test" that tests these in tandem instead of making every test configure every other feature.